integrations/pom.xml

```
@@ -16,7 +16,7 @@
     <jackson.version>2.13.1</jackson.version>
     <enforcer.skip>true</enforcer.skip> <!-- I give up, too much effort to maintain enforcer here, should probably try clear out all enforcer only deps and see if it works -->
     <netty.version>4.1.73.Final</netty.version>
-    <jenkins.version>2.319.3</jenkins.version>
+    <jenkins.version>2.332.1</jenkins.version>
```

should be updated to .1 when released, for now it should be the weekly version
```suggestion
    <jenkins.version>2.332</jenkins.version>
```